demos/gui/src/main/release/etc/config/jgroups-relay1.xml

```diff
@@ -18,57 +18,49 @@
          ucast_send_buf_size="640000"
          mcast_recv_buf_size="25000000"
          mcast_send_buf_size="640000"
-         loopback="true"
          max_bundle_size="64k"
          bundler_type="transfer-queue"
          ip_ttl="${jgroups.udp.ip_ttl:0}"
          enable_diagnostics="true"
          thread_naming_pattern="cl"
 
-         timer_type="new"
          timer.min_threads="2"
          timer.max_threads="4"
          timer.keep_alive_time="3000"
          timer.queue_max_size="1000"
-         timer.wheel_size="200"
-         timer.tick_time="50"
 
-         thread_pool.enabled="true"
          thread_pool.min_threads="2"
          thread_pool.max_threads="8"
          thread_pool.keep_alive_time="5000"
          thread_pool.queue_enabled="true"
          thread_pool.queue_max_size="100000"
          thread_pool.rejection_policy="discard"
 
-         oob_thread_pool.enabled="true"
          oob_thread_pool.min_threads="1"
          oob_thread_pool.max_threads="8"
          oob_thread_pool.keep_alive_time="5000"
          oob_thread_pool.queue_enabled="false"
          oob_thread_pool.queue_max_size="100"
          oob_thread_pool.rejection_policy="discard"/>
 
-    <PING timeout="2000"
-            num_initial_members="3"/>
+    <PING/>
     <MERGE3/>
     <FD_SOCK />
     <FD_ALL />
     <VERIFY_SUSPECT timeout="1500"  />
-    <BARRIER />
-    <pbcast.NAKACK use_mcast_xmit="true"
-                   retransmit_timeout="100,300,600,1200"
+    <pbcast.NAKACK2 use_mcast_xmit="true"
                    discard_delivered_msgs="true"/>
     <UNICAST3 conn_expiry_timeout="0"/>
     <pbcast.STABLE stability_delay="1000" desired_avg_gossip="50000"
                    max_bytes="1000000"/>
     <pbcast.GMS print_local_addr="true" join_timeout="${jgroups.join_timeout:5000}"
                 max_bundling_time="200"
                 view_bundling="true"/>
-    <FC max_credits="2M"
-        min_threshold="0.40"/>
+    <UFC max_credits="2M"
+          min_threshold="0.40"/>
+    <MFC max_credits="2M"
```

Comment:
Wow, I didn't know that. Thanks a lot Bela for clarifying this!

On Mon, Aug 29, 2016 at 1:52 PM, Bela Ban <notifications@github.com> wrote:

> In demos/gui/src/main/release/etc/config/jgroups-relay1.xml
> <https://github.com/infinispan/infinispan/pull/4529#discussion_r76594155>:
>
> >                     discard_delivered_msgs="true"/>
> >      <UNICAST3 conn_expiry_timeout="0"/>
> >      <pbcast.STABLE stability_delay="1000" desired_avg_gossip="50000"
> >                     max_bytes="1000000"/>
> >      <pbcast.GMS print_local_addr="true" join_timeout="${jgroups.join_timeout:5000}"
> >                  max_bundling_time="200"
> >                  view_bundling="true"/>
> > -    <FC max_credits="2M"
> > -        min_threshold="0.40"/>
> > +    <UFC max_credits="2M"
> > +          min_threshold="0.40"/>
> > +    <MFC max_credits="2M"
>
> Ah, no, with UDP you need both UFC and MFC, unless you don't want to have
> flow control. I suggest leave UFC/MFC in; you can still skip flow control
> by setting flag NO_FC *per message*. This makes sense, e.g. if it is a
> blocking sync RPC.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/infinispan/infinispan/pull/4529/files/ab2c227a91676f5091a8c6ee4d6f338c58b88d38#r76594155>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AIBvhunfePrpjNgvS_TL54PXvAXp9taqks5qksgPgaJpZM4Js0bW>
> .
>